addons/crm/models/res_config_settings.py

```diff
@@ -130,23 +130,32 @@ def _compute_predictive_lead_scoring_field_labels(self):
                 setting.predictive_lead_scoring_field_labels = _('Stage')
 
     def set_values(self):
-        group_lead_before = self.env.ref('crm.group_use_lead') in self.env.user.groups_id
+        group_use_lead = self.env.ref('crm.group_use_lead')
```

Comment:
Actually you could xmlid_to_resid directly to further gain queries. This group is a master data anyway.